app/controllers/hosts_controller.rb

```diff
@@ -560,6 +560,7 @@ def process_hostgroup
               Host.new(params[:host])
             end
     @host.set_hostgroup_defaults
+    @host.set_compute_attributes
```

Comment:
Doesn't that call the compute resource controller to refresh the VM tab, rather than process_hostgroup?